### PR TITLE
Signin

### PR DESCRIPTION
**Pull request by flystar0526**
Signin Page

### CODE REVIEW

File: src/pages/auth/Signin.tsx [hunk 1]
```diff
@@ -1,66 +1,189 @@
...
   password: string;
+  checkPassword?: string;
 }
 
 const Signin: React.FunctionComponent = () => {
   const navigate = useNavigate();
-
+  const dispatch = useAppDispatch();
   const [form] = Form.useForm();
+  const [messageApi, contextHolder] = message.useMessage();
```
Comment (xLanStar) [inline src/pages/auth/Signin.tsx:34]:
```suggestion
  const { message } = App.useApp();
```

File: vite.config.ts [hunk 2]
```diff
@@ -10,4 +10,14 @@ export default defineConfig({
...
   ],
+  server: {
+    proxy: {
+      "/api": {
+        target: "http://localhost:5000",
+        changeOrigin: true,
+        rewrite: (path) => path.replace(/^\/api/, ""),
+      },
+    },
+  },
```
Comment (xLanStar) [inline vite.config.ts:21]: This is ok for now. I will replace it by environment variable latter

File: src/pages/auth/Signin.tsx [hunk 3]
```diff
@@ -1,66 +1,189 @@
...
-      (logo)
-      <h3>登入</h3>
-      <Form
-        form={form}
-        layout="vertical"
-        scrollToFirstError
-        onFinish={onFinish}
-        noValidate
+    <Fragment>
+      {contextHolder}
```
Comment (xLanStar) [inline src/pages/auth/Signin.tsx:73]:
```suggestion
```

**Review by xLanStar — CHANGES_REQUESTED** [review]

**Review by LanstarFN — APPROVED** [review]
LGTM

**Review by xLanStar — APPROVED** [review]
LGTM

**Pull request merged**